server/assets/stylesheets/reset-password.css

```diff
@@ -425,7 +425,7 @@ input#email.textInput__suffix {
 button#cancel {
     position: absolute;
     top: 60px;
-    left: 60px;
+    left: 0;
```

Comment:
Looking at staging, I think we should actually remove the `left` setting altogether.

Reply:
I agree, I will be removing this line altogether